src/Feature.php

```diff
@@ -61,23 +66,46 @@ public function match(string $flag, array $context = [], ?array &$matchAttachmen
      */
     public function evaluate(string $flag, array $context = [], callable ...$callbacks): void
     {
-        $evalContext = new EvalContext();
-        $evalContext->setFlagKey($flag);
-        $evalContext->setEntityContext(array_merge($this->globalContext, $context));
-
-        try {
-            $evaluation = $this->evaluator->postEvaluation($evalContext);
-            $variantKey = $evaluation->getVariantKey();
-            $attachment = $evaluation->getVariantAttachment();
-        } catch (ApiException $e) {
-            $variantKey = 'error';
-            $attachment = [];
-        }
+        [$variantKey, $attachment] = $this->performEvaluation($flag, $context);
 
         $callback = $callbacks[$variantKey]
             ?? $callbacks['otherwise']
             ?? fn (?array $attachment) => false;
 
         $callback($attachment);
     }
+
+    /**
+     * @param string $flag
+     * @param array<mixed> $context
+     * @return array<mixed>
+     */
+    private function performEvaluation(string $flag, array $context): array
```

Comment:
Change it to accept an array of flags?
```suggestion
    private function performEvaluation(array $flag, array $context): array
```

Reply:
Hmm. I'm not sure what that would achieve? Bulk evaluation only occurs when tags are set, otherwise, it's actually just evaluating against a single flag _using_ the bulk endpoint.

Reply:
Additionally, from a consumers perspective, you only really want to 'ask' about a single flag at a time